resources/provider/migrations/postgres/32_v0.11.7.down.sql

```diff
@@ -0,0 +1,2 @@
+-- Resource: elbv2.target_groups
+DROP TABLE IF EXISTS aws_elbv2_target_group_target_health_descriptions;
```

Comment:
You can remove all of the migrations as we have removed that in the latest major release (`12.x`)